frontend/ui/widget/configdialog.lua

```diff
@@ -1153,6 +1153,7 @@ function ConfigDialog:onConfigMoreChoose(values, name, event, args, name_text, m
                     curr_values = self.configurable[name]
                 end
                 widget = DoubleSpinWidget:new{
+                    width = name ~= "word_spacing" and math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * 0.6) or nil,
```

Comment:
Can't this be passed somwho via `more_options_param.something` ? Instead of having to hardcode a name here.